tasks/copy_to_glacier_lambda/README.md

```diff
@@ -0,0 +1,172 @@
+## Description
+
+The `copy_to_glacier_lambda` is meant to be deployed as a lambda function that takes a Cumulus message, extracts a list of files, and copies those files from their current storage location into a staging/glacier location.
+
+
+## Build
+
+The following steps assume you are using a current version of Python 3 (`pip` comes with current versions).
+
+```
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt # requirements-dev.txt if you're testing/developing
+```
+
+
+## Test
+
+This example uses `nose`, a package for testing Python projects.
+
+```
+# nose is installed as part of requirements-dev.txt
+nosetests test
```

Comment:
I'd like to avoid using localstack if we can. The test that's currently failing can be fixed with mocking, but I wasn't sure if that was in the scope of this PR so I haven't addressed it.

Good point about linting. I'll add that.

We will probably have to do integration testing manually, but I'm not sure how best to address that in this document. I'll think on that.